website/docs/podman/creating-a-podman-machine.md

```diff
@@ -38,7 +38,10 @@ Consider creating a custom Podman machine to:
    1. **Disk size**:
       Select the disk size.
    1. **Image path (Optional)**:
-      Select a bootable image containing a virtual machine with Podman.
+      Provide a bootable image using one of the following options:
+      - **File**: Use the option to select an image, such as `podman-machine.aarch64.applehv.raw.zst` from your local machine.
+      - **Url**: Use the option to copy an image URL and paste it in this field. For example, `https://github.com/containers/podman/releases/download/v5.2.2/podman-machine.x86_64.applehv.raw.zst`.
```

Comment:
I dont know it would be good providing links like this as it'd be outdated.

Instead shouldn't we provide a link to the release page and say "download the zst?"

Reply:
I made this change. I think we should not specifically say  `downlaod the zst` because the intent is not to download , it is about how to get an image URL that is to be used with this option.